aleph-client/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph_client"
-version = "3.7.2"
+version = "3.8.0"
```

Comment:
As far as I understand those changes break aleph client (unused) functionalities. In this case shouldn't we update major version?